openmrs/apps/clinical/dashboard.json

```diff
@@ -19,7 +19,17 @@
                     "countyDistrict"
                 ]
             },
-            "diagnosis":{  
+            "validationForms-2.0": {
```

Comment:
the control directive name is validationForms-2.0? can't we find a better name?
how about "validationForms2"?
What is the purpose of the control? if the purpose is to show the forms2 forms, then why not just "forms2Control" or something appropriate?

Reply:
Renaming to Forms2Validations